<?xml version="1.0" encoding="UTF-8"?>
<!-- Generated by org.testng.reporters.JUnitXMLReporter -->
<testsuite ignored="0" hostname="DESKTOP-HADT1D5" failures="3" tests="4" name="Smoke(failed)" time="0.091" errors="0" timestamp="2023-01-22T21:03:47 IST">
  <testcase classname="jan_21.SkipTest" name="login" time="0.019">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at jan_21.SkipTest.login(SkipTest.java:11)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- login -->
  <testcase classname="jan_21.SkipTest" name="logout" time="0.0">
    <skipped/>
  </testcase> <!-- logout -->
  <testcase classname="jan_21.SoftAssertDemo" name="test1" time="0.001">
    <failure type="java.lang.AssertionError" message="expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: expected [true] but found [false]
at jan_21.SoftAssertDemo.test1(SoftAssertDemo.java:13)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 31 stack frames]]>
    </failure>
  </testcase> <!-- test1 -->
  <testcase classname="jan_21.SoftAssertDemo" name="test2" time="0.006">
    <failure type="java.lang.AssertionError" message="The following asserts failed:
	expected [true] but found [false]">
      <![CDATA[java.lang.AssertionError: The following asserts failed:
expected [true] but found [false]
at jan_21.SoftAssertDemo.test2(SoftAssertDemo.java:22)
at java.base/java.util.ArrayList.forEach(ArrayList.java:1511)
... Removed 29 stack frames]]>
    </failure>
  </testcase> <!-- test2 -->
</testsuite> <!-- Smoke(failed) -->
